### ISSUE TITLE
Internationalization

### ISSUE THREAD

**Issue by lucasbrelaz** [ShankyTiwari/ng-material-multilevel-menu#20]
Do you have any forecast of when there will be an option for internationalization?

**Comment by ShankyTiwari** [comment]
Hi, I think it should work with multilingual Data. Also, this [thread](https://github.com/ShankyTiwari/ng-material-multilevel-menu/issues/8) may help you.
May I know which module you are using for Internationalization?

Thanks

**Comment by lucasbrelaz** [comment]
Hi, the internationalization works however when we change the language we need to reload the page to reload the translated data. Using NGX Translate.

**Comment by ShankyTiwari** [comment]
True. When the user changes the language you can reload the menu or replace its labels by passing an updated array with new labels.
